### PR TITLE
Added Oculus VR, Rue du Commerce and Rumble.

### PR DESCRIPTION
**Pull request by alrjunior**
Added Oculus VR (easy), Rue du Commerce (impossible) and Rumble (hard). And put Tiktok and Spotify as a popular sites. They're a popular sites, right?

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -11049,6 +11079,30 @@
...
+        ]
+    },
+
+    {
+        "name": "Rumble",
+        "url": "https://rumblefaq.groovehq.com/help/how-to-delete-your-account",
+        "difficulty": "hard",
+        "notes": "You must send an e-mail to support@rumble.com with the subject \"Delete my account\".",
+        "notes_pt_br": "Você tem que enviar um e-mail para support@rumble.com com o assunto \"Delete my account\".",
+        "email": "support@rumble.com",
```
Comment (tupaschoal) [inline _data/sites.json:11100]:
```suggestion
        "notes": "You must send an to them",
        "notes_pt_br": "Você tem que enviar um e-mail para eles",
        "email": "support@rumble.com",
        "email_subject": "Delete my account",
```

**Review by tupaschoal — COMMENTED** [review]
>  And put Tiktok and Spotify as a popular sites. They're a popular sites, right?

Yeah, we don't pay much attention to those as I believe most people use the website with the search.

I see you did delete "HighBeam Research". Do you have data showing that the service has closed?

Also, did you test the https websites?

**Comment by alrjunior** [comment]
> > And put Tiktok and Spotify as a popular sites. They're a popular sites, right?
> 
> Yeah, we don't pay much attention to those as I believe most people use the website with the search.
> 
> I see you did delete "HighBeam Research". Do you have data showing that the service has closed?
> 
> Also, did you test the https websites?

About the Highbeam Research, there's the wikipedia article about the service - https://en.wikipedia.org/wiki/HighBeam_Research - R.I.P.

And about the HTTPS, I took care of checking all of this links. And all of them exist in the HTTPS version.

**Comment by alrjunior** [comment]
https://en.wikipedia.org/wiki/IBM_Cloud_Video
https://history-computer.com/ustream-complete-guide/

About the Ustream.tv (now, IBM Cloud Video)

**Comment by alrjunior** [comment]
One more correction: IBM Watson Media.

**Pull request merged**